crates/uv-git/src/source.rs

```diff
@@ -120,7 +120,7 @@ impl GitSource {
         // Report the checkout operation to the reporter.
         if let Some(task) = task {
             if let Some(reporter) = self.reporter.as_ref() {
-                reporter.on_checkout_complete(remote.url(), short_id.as_str(), task);
+                reporter.on_checkout_complete(remote.url(), actual_rev.as_str(), task);
```

Comment:
This change avoids the start messages showing the long hash, while the finish message shows the short hash.